subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -1610,6 +1675,10 @@ static isoal_status_t isoal_tx_pdu_emit(const struct isoal_source *source_ctx,
 	/* Attempt to enqueue the node towards the LL */
 	status = source_ctx->session.pdu_emit(node_tx, handle);
 
+	ISOAL_LOG_DBG("[%p] PDU %llu err=%X len=%u frags=%u released",
+		      source_ctx, payload_number, status,
+		      produced_pdu->contents.pdu->len, sdu_fragments);
```

Comment:
Outside the scope of the PR, but what's the deal with `ISOAL_LOG_DBG`? It's defined as
```c
#define ISOAL_LOG_DBG(...)     LOG_DBG(__VA_ARGS__)

#if defined(ISOAL_DEBUG_VERBOSE)
#define ISOAL_LOG_DBGV(...)    LOG_DBG(__VA_ARGS__)
#else
#define ISOAL_LOG_DBGV(...)    (void) 0
#endif /* ISOAL_DEBUG_VERBOSE */
```

So it does nothing unless `ISOAL_DEBUG_VERBOSE` is set, but why not just use it `LOG_DBG`?

Reply:
I'm not certain if I understood the comment correctly, but just to point out that there are two macros there. `ISOAL_LOG_DBG()` and `ISOAL_LOG_DBGV()` with the latter being the verbose version. The standard logging does not provide tiers to control verbosity on debug statements. The verbose statements are useful when debugging the unit tests, but probably a disruption if logging is enable elsewhere.

Reply:
Ah, I misread; didn't see the `V` in `ISOAL_LOG_DBGV`. Arguably that should have been called `ISOAL_LOG_DBG_VERBOSE` to avoid the confusion (i.e. ironically make the macro more verbose). 

Have you considered moving `ISOAL_DEBUG_VERBOSE` to a Kconfig option, similar to https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/host/Kconfig#L952 ?

Reply:
I did consider a longer macro name, but ideally I would have liked these statements to be one liners. A longer macro name means less space for the statement. If there are strong objections let me know otherwise I would like to keep it short.

Thanks for the example kconfig. Let me consider that.

Reply:
Done now.